use-cases/customer_churn/0_cust_churn_overview_dw.ipynb

```diff
@@ -164,7 +164,8 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    !aws s3 cp s3://sagemaker-sample-files/datasets/tabular/customer-churn/customer-churn-data.zip ./data/raw
</pre><p>This line didn't work for me. It copied the file into a file called <code>raw</code> .</p><p>Please change to</p><pre class="ql-syntax" spellcheck="false">!mkdir -p data/raw/
!aws s3 cp s3://sagemaker-sample-files/datasets/tabular/customer-churn/customer-churn-data.zip ./data/raw/
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2817/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>